pkg/types/imagebased/imagebased_config_types.go

```diff
@@ -0,0 +1,109 @@
+	// It defaults to -40G (40GB before the end of the disk).
+	// +optional
+	ExtraPartitionStart string `json:"extraPartitionStart,omitempty"`
+
+	// ImageContentSources lists sources/repositories for the release-image content.
+	// The field is deprecated. Please use imageDigestSources.
+	// +optional
+	DeprecatedImageContentSources []types.ImageContentSource `json:"imageContentSources,omitempty"`
+
+	// IgnitionConfigOverride is a JSON string containing the user overrides for
+	// the Ignition configuration. It will be merged with the ignition config
+	// generated by the image-based installer. Supported Ignition version is 3.2.
+	// +optional
+	IgnitionConfigOverride string `json:"ignitionConfigOverride,omitempty"`
+
+	// ImageDigestSources lists sources/repositories for the release-image content.
+	// +optional
+	ImageDigestSources []types.ImageDigestSource `json:"imageDigestSources,omitempty"`
+
+	// InstallationDisk is the disk used to install the seed image. Provide the
+	// device by ID, e.g. /dev/by-id/ata-xxxxx
+	InstallationDisk string `json:"installationDisk"`
+
+	// NetworkConfig is a YAML manifest that can be processed by nmstate, using
+	// custom marshaling/unmarshaling that will allow to populate nmstate config
+	// as plain yaml.
+	// +optional
+	NetworkConfig aiv1beta1.NetConfig `json:"networkConfig,omitempty"`
+
+	// PrecacheBestEffort is a flag to enable best effort precaching.
+	// +optional
+	PrecacheBestEffort bool `json:"precacheBestEffort,omitempty"`
+
+	// PrecacheDisabled is a flag to disable precaching.
+	// +optional
+	PrecacheDisabled bool `json:"precacheDisabled,omitempty"`
+
+	// Proxy defines the proxy settings for the cluster.
+	// If unset, the cluster will not be configured to use a proxy.
+	// +optional
+	Proxy *types.Proxy `json:"proxy,omitempty"`
+
+	// PullSecret is the secret to use when pulling images.
+	PullSecret string `json:"pullSecret"`
+
+	// SeedImage is the seed image to use for the installation. This image will be
+	// used to prepare the installation disk.
+	SeedImage string `json:"seedImage"`
+
+	// SeedVersion is the version of the seed image. It will be validated against
+	// the seed image version.
+	SeedVersion string `json:"seedVersion"`
+
+	// Shutdown is a flag to shutdown the host after installation. It defaults to
+	// false.
+	// +optional
+	Shutdown bool `json:"shutdown,omitempty"`
+
+	// SkipDiskCleanup is a flag to skip disk cleanup before installation. As part
+	// of installation we will try to format the disk this flag will skip that
+	// step. It defaults to false.
+	SkipDiskCleanup bool `json:"skipDiskCleanup,omitempty"`
+
+	// SSHKey is the public Secure Shell (SSH) key to provide access to instances.
+	SSHKey string `json:"sshKey,omitempty"`
+
+	// UseContainersFolder is a flag to use /var/lib/containers as a folder and not
+	// an extra partition. Defaults to false.
+	UseContainersFolder bool `json:"useContainersFolder,omitempty"`
```

Comment:
Lets remove it, i assume we should remove it from lca too, ocp doesn't support using folders for now